content/docs/16-component.md

```diff
@@ -52,6 +58,8 @@ This page lists the version details of various Palette components with respect t
 
 |Palette Release|On-Premises Installer Version|
```

Comment:
```suggestion
|Palette Release|On-Prem Installer Version|
```

Reply:
Per our word list, change On-Premises to On-Prem in section title and table header.